File: ApiSuvesaPos/Negocio/Modelo/inventario/pretomageneral/pretomageneral.vb
Namespace Modelo
    Public Class pretomageneral
        Public Property IdPreToma As Integer
        Public Property Fecha As Date
        Public Property Anulado As Boolean
        Public Property Aplicado As Boolean
        Public Property Encargado As String
        Public Property PreTomaGeneralDetalle As New List(Of Modelo.pretomageneraldetalle)
    End Class
End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/inventario/pretomageneral/pretomageneral.cs
namespace Modelo
{
    public class pretomageneral
    {
        public int IdPreToma { get; set; }
        public DateTime Fecha { get; set; }
        public bool Anulado { get; set; }
        public bool Aplicado { get; set; }
        public string Encargado { get; set; }
        public List<Modelo.pretomageneraldetalle> PreTomaGeneralDetalle { get; set; } = new List<Modelo.pretomageneraldetalle>();
    }
}
